### DISCUSSION TITLE
Digit not working.

### DISCUSSION THREAD

**Discussion by acr1161** [simpleavr/s14clock#13] [Q&A] [answered]
The second to last right side digit will not light. Checked solder and all appears correct.

**Comment by simpleavr** [comment, marked as answer]
I suggest check soldering again, not just the PCB, it could also be the S2 Mini soldering joints.
Do you have V2 (smd chip) or V3 (no smd chip) hardware? I can locate the IO pin you need to check if you indicate the HW version.

If you have a jumper wire, you can also verify by shorting D5 to any other D? digit pin (within the same LED module, PCB has identifier marks on pin mapping) and see if light up. This will confirm whether it is a deflective unit.

It can also be a deflective S2 Mini unit.

If you run through testing and confirm it is a deflective LED module or S2 Mini. I can send you a replacement LED module to get it replaced. If you don't have desoldering tools or not comfortable with desoldering, I can replace your whole unit with a working one. I do need you to send in the deflective unit so I can recover it.

Thanks.

**Reply by acr1161** [reply]
V3. Dated may 2023 v3

**Reply by acr1161** [reply]
Found it. Bad solder connection on mini board. Reflowed joint. Digit now works. 

Thank you v

**Reply by acr1161** [reply]
Thank you

Allan.
